packages/block-library/src/embed/core-embeds.js

```diff
@@ -137,13 +137,17 @@ export const others = [
 		patterns: [ /^https?:\/\/cloudup\.com\/.+/i ],
 	},
 	{
+		// Deprecated since CollegeHumor content is now powered by YouTube
 		name: 'core-embed/collegehumor',
 		settings: {
 			title: 'CollegeHumor',
 			icon: embedVideoIcon,
 			description: __( 'Embed CollegeHumor content.' ),
+			supports: {
+				inserter: false,
+			}
 		},
-		patterns: [ /^https?:\/\/(www\.)?collegehumor\.com\/.+/i ],
+		patterns: [ /^https?:\/\/(www\.)?collegehumor\.com\/.+/i, /^https?:\/\/((m|www)\.)?youtube\.com\/.+/i, /^https?:\/\/youtu\.be\/.+/i ],
```

Comment:
I would expect this array to become empty from now on to avoid matching those patterns with this particular embed. When you paste the link in the empty Paragraph block, it will probably transform automatically. I'm not sure what will happen with YouTube links in this case as well.